### ISSUE TITLE
Rebalancing Salt and Saltwater

### ISSUE THREAD

**Issue by Mecares** [CleverRaven/Cataclysm-DDA#14854]
In its current state the quench values for salt and saltwater are highly unrealistic, ~~also the amount of salt crafted from evaporating salt water is much higher than the salinity of ocean water, which i consider to be a sensible baseline for what salt water is.~~

Evaporating a liter (or a kg) of ocean water will result in 30g to 40g of various salts most of those are sodium chloride which we normally call salt.

The amount of salt currently represented by one unit of salt is enough to kill  a human with a body weight of 60kg. (The LD50 of sodium chloride is 1g of salt per kg of body weight for humans, which means it will result with 50% chance in death if consumed.) [other sources state higher values like 3g/kg for rats but those are hard to kill]

Consuming 1,5 times the LD50 is what i arbitrarily consider certainly deadly, with a survivor of a body weight of 80kg we would consider 120g of salt deadly. With a deadly quench value of 1200 we get a convenient ratio of 10 quench per 1g of salt.

This would result with the current salt portion size in an realistic quench value of -600 per portion for salt and -550 for salt water (-600 plus the 50 quench of regular water).

Since these numbers are obviously insanely high i would suggest to change the size of one portion of salt from 60g to 8g which will be the realistic amount of salt resulting out of evaporating 0,25 liters of saltwater (one volume unit). Which will result in more sane values of -80 quench for salt and -30 quench for saltwater (the current value). Currently pure salt makes you less thirsty than the same amount of salt solved in water. This would also makes several recipes for food more realistic, for example nachos currently require 120g of salt (2 portions) which is enough to certainly kill most persons.

Volume: a unit of volume equals around 250 cm^3 with a density of 2.16g and an assumed porosity of 30% we would get  378g of salt per volume unit. With a portion size of 8g we would have a volume per portion of 0.021.

TL:DR
Suggestion: Change the volume of one portion of salt to 0.021, weight to 0.008 kg and quench to -80 and for salt water weight to 0.258 kg.

The obvious disadvantage is that ~~several~~ *some* recipes especially those to craft salt and non-cooking recipes that use salt for example tanning would also have to be re-balanced.

/edit: Most cooking recipes already use 1 volume of saltwater equally to one volume of salt, which would fit quite well to the suggested change.

/edit2: It would also correct the weird mechanic that you can craft 1 salt water with 1 salt but need 8 salt water to craft one salt.

**Comment by Zireael07** [comment]
60 g of salt is insanely high indeed. Wonder how that never got noticed.

**Comment by Coolthulhu** [comment]
> This would result with the current salt portion size in an realistic quench value of -600 per portion for salt and -550 for salt water (-600 plus the 50 quench of regular water).

Quench values for salt and salt water aren't worth rebalancing anything around them.
There is no in game mechanic that makes use of that, it is not good design to have those even be edible.

> This would also makes several recipes for food more realistic, for example nachos currently require 120g of salt (2 portions) which is enough to certainly kill most persons.

This one makes sense - way too much ordinary salt is used for stuff.
Many recipes call for an entire unit of salt or just 5 units of seasoned salt (100 seasoned salt is made from 2 salt, 1 sugar and 20 pepper).
Nachos are probably quite old, only other food recipes that use 2 salt are tacos, seasoned salt and italian seasoning.

**Comment by SeanMirrsen** [comment]
There is, actually, a good reason to keep saltwater drinkable. Isn't there a CBM that automatically filters water? (If there isn't, there probably should be.) Saltwater should remain drinkable so that it could be consumed by someone with such a CBM.
It's a better argument to make salt itself non-edible, but then you have to both make it non-comestible, and still let it be food for sorting purposes. And if there ever is a set of horse-themed mutations, salt will be a vital daily consumable on par with vitamins.

I agree entirely with the rest of the OP.

**Comment by Zireael07** [comment]
Salt itself could remain edible if it worked to counter food poisoning.

**Comment by Coolthulhu** [comment]
> Isn't there a CBM that automatically filters water?

Nope. And even if it existed, it would be better to have it work like internal furnace than to pollute everyone's menus with salt water that they almost certainly don't want to drink, by accident or otherwise.
> you have to both make it non-comestible, and still let it be food for sorting purposes

Not necessarily. Bones, pinecones, pine boughs, meds etc. are not sorted as food. Salt isn't really edible so sorting it as food only makes sense for inventory shuffling.

Keeping salty all this pseudo-comestible stuff edible is simply bad design.

**Comment by SeanMirrsen** [comment]
Arguably, it's worse design to sort items into game-logic categories, in a game keyed to real-logic mechanics. I mean. You *can* eat salt. Salt is a thing that can be eaten. Granted, not by itself, but salt is an edible product, so it makes sense that it has comestible values.

While you're out there correcting bad design, why not move the two-by-four and the pipe out of the "weapons" category? Along with other things that are usable for hitting things, but are not explicitly weapons? Because their primary ingame use is construction, not combat.

**Comment by illi-kun** [comment]
> While you're out there correcting bad design, why not move the two-by-four and the pipe out of the "weapons" category? Along with other things that are usable for hitting things, but are not explicitly weapons?

It will be done sooner or later.

**Comment by Coolthulhu** [comment]
Created an issue for the whole category thing (including pipes and two-by-fours): #14855

**Comment by Headjack** [comment]
I think salt should remain edible, along with bleach and ammonia and the like. A lot of roguelikes have that sort of, 'what if I do this' feeling to them, and I remember back when I began in Cata having fun trying to eat different things, throwing stuff at zombies, and make a gas bomb with the chemicals. Occasionally I hit the wrong key and eat a seed or some salt or whatever, but that's not bad mechanics, it's bad controls. I usually pass it off by saying my character had an accident, heh.

**Comment by Coolthulhu** [comment]
> A lot of roguelikes have that sort of, 'what if I do this' feeling to them

But DDA clearly says what will happen.
Because seriously, why would anyone drink bleach?
> Occasionally I hit the wrong key and eat a seed or some salt or whatever, but that's not bad mechanics, it's bad controls.

The best way to fix the controls here is to fix the mechanics.

How would you fix it differently?
By asking "are you sure you want to do this stupid thing?" before eating salt? This would still remove the experimenting part by informing the player that the choice is obviously bad.

**Comment by Headjack** [comment]
I think it's a symptom of the keyboard interface more than anything, not really something demanding a fix. When I say bad controls, I mean a somewhat primitive interface which we're sort of tied to.

Letting the player make disadvantageous choices for laughs is fine, in my mind. There's things like eating paper for some small nutrition, or burning valuables for fuel or CBM power. It's no good to just adjudicate what's smart or dumb via mechanics.

**Comment by sparr** [comment]
> Because seriously, why would anyone drink bleach?

Because I'm vomiting black goo and considering last resorts.
